pkg/components/prometheus-operator/component_conversion_test.go

```diff
@@ -0,0 +1,280 @@
+package prometheus
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/hcl/v2"
+	"gopkg.in/yaml.v2"
+
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+)
+
+func renderManifests(t *testing.T, configHCL string) map[string]string {
+	name := "prometheus-operator"
+
+	component := newComponent()
+
+	body, diagnostics := util.GetComponentBody(configHCL, name)
+	if diagnostics != nil {
```

Comment:
Why not use `HasErrors()` here?

Reply:
Why?

Reply:
We can use, this part of the code kinda reflected code from elsewhere.

Reply:
We need to make that consistent. I suggest we use `HasErrors()` for now.

Reply:
Created some issues for it.